src/intl/vi/common.json

```diff
@@ -122,10 +126,10 @@
   "individuals": "Cá nhân",
   "jobs": "Công việc",
   "kraken-logo": "Logo của Kraken",
-  "language-am": "Tiếng Amhara",
-  "language-be": "Tiếng Belarus",
+  "langauge-am": "Tiếng Amhara",
   "language-ar": "Tiếng Ả Rập",
   "language-az": "Tiếng Azerbaijani",
+  "langauge-be": "Tiếng Belarus",
```

Comment:
`
  "language-am": "Tiếng Amhara",
  "language-ar": "Tiếng Ả Rập",
  "language-az": "Tiếng Azerbaijani",
  "language-be": "Tiếng Belarus",
`